app/component/front-page/NearestRoutesContainer.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component, PropTypes } from 'react';
+import Relay, { Route } from 'react-relay';
+import NearbyRouteListContainer from '../route/NearbyRouteListContainer';
+
+class NearbyRouteListContainerRoute extends Route {
+  static queries = {
+    stops: (RelayComponent, variables) => Relay.QL`
+      query {
+        viewer {
+          ${RelayComponent.getFragment('stops', variables)}
+        }
+      }
+    `,
+  };
+  static paramDefinitions = {
+    lat: { required: true },
+    lon: { required: true },
+    currentTime: { required: true },
+  };
+  static routeName = 'NearbyRouteListContainerRoute';
+}
+
+export default class NearestRoutesContainer extends Component {
+  static propTypes = {
+    lat: PropTypes.number.isRequired,
+    lon: PropTypes.number.isRequired,
+    currentTime: PropTypes.number.isRequired,
+    modes: PropTypes.array.isRequired,
+  };
+
+  constructor() {
+    super();
+    this.useSpinner = true;
+  }
+
+  componentDidMount() {
+    this.useSpinner = false;
```

Comment:
Works as before. See [this](https://github.com/HSLdevcom/digitransit-ui/pull/584/files#diff-86eaa8a44fc413fd1b866b4c942534bbL11) for previous version. It is done in order to show the spinner on initial load, but not when updating data every 30 seconds.